account/user_blackbox_test.go

```diff
@@ -125,6 +127,7 @@ func createAndLoadUser(s *userBlackBoxTest) *account.User {
 		ImageURL: "someImageUrl" + uuid.NewV4().String(),
 		Bio:      "somebio" + uuid.NewV4().String(),
 		URL:      "someurl" + uuid.NewV4().String(),
+		Cluster:  "somecluster" + uuid.NewV4().String(),
```

Comment:
Let's also add tests when we create or save a user without cluster then it fails (the cluster can't be null or empty)

Reply:
e13d57d71788b85bfa31808932b0554573218ffd